src/components/containers/TraceAccordion.js

```diff
@@ -59,63 +81,72 @@ class TraceAccordion extends Component {
       );
     });
 
-    return Object.keys(fullDataArrayPositionsByTraceType).map((type, index) => {
-      return (
-        <TraceFold
-          key={index}
-          traceIndexes={dataArrayPositionsByTraceType[type]}
-          name={traceTypes(_).find(t => t.value === type).label}
-          fullDataArrayPosition={fullDataArrayPositionsByTraceType[type]}
-        >
-          {this.props.children}
-        </TraceFold>
-      );
-    });
+    return Object.keys(fullDataArrayPositionsByTraceType).map((type, index) => (
+      <TraceFold
+        key={index}
+        traceIndexes={dataArrayPositionsByTraceType[type]}
+        name={traceTypes(_).find(t => t.value === type).label}
+        fullDataArrayPosition={fullDataArrayPositionsByTraceType[type]}
+      >
+        {this.props.children}
+      </TraceFold>
+    ));
   }
 
   renderUngroupedTraceFolds() {
-    if (!this.filteredTraces.length) {
-      return null;
-    }
-
-    return this.filteredTraces.map((d, i) => {
-      return (
+    return (
+      this.filteredTraces.length > 0 &&
```

Comment:
I think this changes the meaning... now when `length === 0` we return `false` instead of `null`.

Reply:
Yeah, but AFAIK behaviour stays the same

Reply:
could we return null here if it doesn't meet this condition.

Reply:
what difference would that make?

Reply:
If `null` and `false` are both ignored by React then I'm fine with this. If React also ignores `[]` then we can just return the result of `.map()` without the guard.